mcs/class/System/System.Net.Sockets/Socket.cs

```diff
@@ -1118,8 +1118,14 @@ static SafeSocketHandle Accept_internal (SafeSocketHandle safeHandle, out int er
 
 #region Bind
 
+#if FEATURE_NO_BSD_SOCKETS
+		[Obsolete  ("System.Net.Sockets.Socket:Bind is not supported on this platform.")]
+#endif
```

Comment:
I disagree. Developers using netstandard or PCL won't see them. Only the ones using the old approach to have dll for every platform would benefit from this but that's the approach which we are no longer promoting. So in the end one would get waring the other one not compiling/running exactly same code which I consider inconsistent.